### ISSUE TITLE
Watch Expressions Don't Work in Chrome Remote Debugging

### ISSUE THREAD

**Issue by mitchell-as** [Komodo/KomodoEdit#1785]
### Short Summary

Title.

### Steps to Reproduce

```
<html>
<head>
    <script type="text/javascript">
        function foo() {
            var x = 1;
            var y = 2;
            console.log(x + y);
        }
    </script>
</head>
<body>
    <h1 onclick="foo();">HELLO!!!</h1>
</body>
```

1. Place a breakpoint on the `var x = 1` line and start debugging.
2. Click HELLO!!!.
3. Switch to the watch pane and click `+` and add the expression `x + y`.
4. Step through a couple of times.
5. At the `console.log()` line, `x + y` is still `Unknown property: x + y`.

When debugging in Python this works.

### Expected results

`x + y` watch expression should be `3` eventually.

### Actual results

It's still `Unknown property`.

### Platform Information

*Komodo Edit or IDE?* IDE
*Komodo Version?* X
*Operating System (and version)?* All

### Additional Information

```
DEBUG:chrome.dbgp.DBGPClient:Waiting for commands from IDE...
DEBUG:chrome.dbgp.DBGPClient:Received data from IDE: 'property_get -c 0 -d 0 -n "x + y" -i 41'
DEBUG:chrome.dbgp.DBGPClient:Processing command 'property_get -c 0 -d 0 -n "x + y" -i 41'
DEBUG:chrome.dbgp.DBGPClient:Fetched 3 properties.
DEBUG:chrome.dbgp.DBGPClient:Unknown property 'x + y'
```

**Comment by mitchell-as** [comment]
Fixed via https://github.com/Komodo/KomodoIDE/commit/40d4e64aa98fd24237e4601a103f4948ed474ecd

**Comment by cgchoffman** [comment]
@mitchell-as just want to make sure that this fix will resolve #1796 as well.

**Comment by mitchell-as** [comment]
This issue and #1796 are two separate issues. This one has been fixed.
